### PR TITLE
492 user feedback on documentation

### PR DESCRIPTION
**Pull request by chlebowa**
Closes [this issue](https://github.com/insightsengineering/coredev-tasks/issues/492)

### CODE REVIEW

**Review by gogonzo — APPROVED** [review]
👍 Precise cut

File: vignettes/including-data-in-teal-applications.Rmd [hunk 1]
```diff
@@ -161,10 +175,10 @@ The filter panel supports MAEs out of the box.
...
 ##### `datanames`
 
-The `datanames` property lists the objects stored in the `teal_data` that constitute datasets of interest.
-Objects passed to `teal_data` become automatically listed in the `datanames` property of the resulting object.
-Objects created in `teal_data` by evaluating code need not be data objects of interest and as such they are not automatically added to `datanames`.
-Use the `datanames` function to modify the `datanames` property.
+The `datanames` property lists the variables stored in the `teal_data` object that constitute datasets of interest.
+Variables passed to `teal_data` become automatically listed in the `datanames` property of the resulting object.
+Variables created in `teal_data` by evaluating code need not be data objects of interest and as such they are not automatically added to `datanames`.
+`datanames` can be read or modified with the `datanames` function.
```
Comment (lcd2yyz) [inline vignettes/including-data-in-teal-applications.Rmd:181]: The original wording make more sense to me. Referring to the code example, `iris` and `cars` are dataset objects, they are not "variables". Variables are always referring to the columns within a dataset, `iris$Sepal.Length` is a variable.
Suggesting to use "data objects" or "dataset objects" instead.

File: vignettes/including-data-in-teal-applications.Rmd [hunk 2]
```diff
@@ -180,8 +194,12 @@ datanames(data_with_code) <- c("iris", "cars")
 datanames(data_with_code)
 ```
 
-All `teal` modules take a `datanames` argument that determines which datasets they are to have access to.
-Only objects enumerated in the `datanames` property of the `teal_data` object can be used.
+The `datanames` property serves as a communication bridge between the data container and modules in a `teal` application.
+In `teal` all modules are called with a `datanames` argument that determines which of the variables in the `teal_data` object they are to access.
+Only variables enumerated in the `datanames` property are eligible for use in modules.
```
Comment (lcd2yyz) [inline vignettes/including-data-in-teal-applications.Rmd:199]: Revise reference to "variables", same as above.

**Review by lcd2yyz — COMMENTED** [review]
General comment: vignettes would benefit greatly from another round of proof-reading, not to revise the contents, but for example, rewording, grammar checks, adding commas, etc. to improve readability

**Pull request merged**